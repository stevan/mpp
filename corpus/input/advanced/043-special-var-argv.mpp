for my $arg (@ARGV) {
    print($arg);
}