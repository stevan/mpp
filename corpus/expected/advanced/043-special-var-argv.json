[
  {
    "type": "Foreach",
    "variable": {
      "type": "Variable",
      "name": "$arg"
    },
    "declarator": "my",
    "listExpr": {
      "type": "Variable",
      "name": "@ARGV"
    },
    "block": [
      {
        "type": "Call",
        "name": "print",
        "arguments": [
          {
            "type": "Variable",
            "name": "$arg"
          }
        ]
      }
    ]
  }
]